pandas/tests/groupby/test_groupby_iloc.py

```diff
@@ -0,0 +1,154 @@
+""" Test positional grouped indexing with iloc GH#42864"""
```

Comment:
just `test_iloc.py` (its already in the groupby directory)

Reply:
As my comment above, but I will change it.